cdap-ui/app/cdap/components/PluginJSONCreator/Create/Content/PluginInput/index.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import WidgetWrapper from 'components/ConfigurationGroup/WidgetWrapper';
+import * as React from 'react';
+
+const PluginInput = ({
+  widgetType,
+  setValue,
```

Comment:
nit: This should be `onChange`. It is being passed to the `onChange` prop of `WidgetWrapper` but this is named as `setValue`. `value` and `onChange` are standard for all `AbstractWidgets`

Reply:
changed!